changelogs/fragments/63280-fix_acl_spaces_in_path.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - acl - fixed module failure if there are spaces in a path
```

Comment:
what i got while parsing, though it might work in changelog building, I have not looked at that system

`"Syntax Error while loading YAML.\n  could not find expected ':'\n\nThe error appears to be in '/home/bcoca/work/ansible/lib/ansible/test.yml': line 3, column 1, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\n(specified line no longer in file, maybe it changed?)"`

Reply:
Yeah, @abadger is right, there're plenty of files in `changelogs/fragments` without a second dash being escaped. Nonetheless, let's keep it robust, I'll fix the line.

Reply:
Done :)

Reply:
@bcoca Not sure what you're doing.  pyyaml handles it fine:

```
$ cat test.yml                    *[temporary-2.9.1-branch-releng-only]  (10:36:55)
test:
  - one - two
[pts/0@peru /srv/ansible/vanilla]$ python3                         *[temporary-2.9.1-branch-releng-only]  (10:36:09)
Python 3.7.4 (default, Jul  9 2019, 16:48:28) 
[GCC 8.3.1 20190223 (Red Hat 8.3.1-2)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import yaml
>>> yaml.__version__
'5.1'
>>> yaml.safe_load(open('test.yml'))
{'test': ['one - two']}
```